styles.css

```diff
@@ -216,7 +217,9 @@ button:hover {
 }
 
 .project-check button {
+  font-family: 'Poppins', sans-serif;
   background: transparent;
+  background-color: white;
```

Comment:
- [ ] Please 🙏🏽 implement the button interactions (enable, hover, etc.).

- The last reviewer suggested 👀 :

> Kindly, I suggest you apply some kind of interaction when using the buttons (normal and hover) original design of Figma.

![buttons](https://user-images.githubusercontent.com/58642949/186332950-39239aa8-9962-416a-b54e-8aebd8efcd84.png)